src/graphics/program-lib/standard.js

```diff
@@ -401,6 +413,7 @@ pc.programlib.standard = {
 
         code += varyings;
         code += chunks.basePS;
+        code += chunks.textureBordersPS;
```

Comment:
But it never gets called in non-UI shaders – will it have any effects post-linking?